core/modules/cart/store/actions/methodsActions.ts

```diff
@@ -60,7 +60,7 @@ const methodsActions = {
     }
   },
   async updateShippingMethods ({ dispatch }, { shippingMethods }) {
-    if (shippingMethods.length > 0) {
+    if (shippingMethods && Array.isArray(shippingMethods) && shippingMethods.length > 0) {
```

Comment:
Can you make it shorter?
`if (Array.isArray(shippingMethods) && shippingMethods.length)`